src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -511,24 +510,24 @@ def reset_service_principal_credential(name, secret=None, years=1):
 
     client.applications.patch(app.object_id, app_create_param)
 
-    _build_output_content(name, sp_object_id, secret, client.config.tenant_id)
+    _build_output_content(name, secret, client.config.tenant_id)
+
+def _build_output_content(sp_name, secret, tenant):
+    logger.warning("Service principal has been configured.")
+    logger.warning("  id(client_id):           " + sp_name)
+    logger.warning("  password(client_secret): " + secret)
 
-def _build_output_content(sp_name, sp_object_id, secret, tenant):
-    logger.warning("Service principal has been configured with name: '%s', secret: '%s'",
-                   sp_name, secret)
     logger.warning('Useful commands to manage azure:')
     logger.warning('Assign a role:')
-    logger.warning('    az role assignment create --assignee %s --role Contributor', sp_object_id)
+    logger.warning('    az role assignment create --assignee %s --role Contributor', sp_name)
     logger.warning('Log in:')
     logger.warning('    az login --service-principal -u %s -p %s --tenant %s',
                    sp_name, secret, tenant)
     logger.warning('Reset credentials:')
-    logger.warning('    az ad sp reset-sp-credentials --name %s', sp_name)
+    logger.warning('    az ad sp reset-credentials --name %s', sp_name)
     logger.warning('Revoke:')
-    logger.warning('    az ad sp delete --id %s', sp_name)
```

Comment:
it is by design that in the developer tenant (which is our case), that deleting the application will cause service principal object to be deleted automatically. So we don't need this.